server/etcdserver/server.go

```diff
@@ -2202,10 +2231,6 @@ func (s *EtcdServer) compactRaftLog(snapi uint64) {
 		}
 		lg.Panic("failed to compact", zap.Error(err))
 	}
-	lg.Info(
-		"compacted Raft logs",
-		zap.Uint64("compact-index", compacti),
-	)
```

Comment:
why remove this log?

Reply:
Now happens too frequently and spams logs

Reply:
Done